nautobot_chatops/workers/nautobot.py

```diff
@@ -195,7 +211,7 @@ def get_vlans(dispatcher, filter_type, filter_value_1):
     if not filter_type:
         prompt_for_vlan_filter_type("nautobot get-vlans", "select a vlan filter", dispatcher)
         return False
-    if not filter_value_1:
+    if not filter_value_1 or is_menu_track_item(filter_value_1):
```

Comment:
A general nitpick - I find logic of the form `if not x or y` to be a bit non-obvious. Would you be able to change these throughout to `if y or not x` (such as `if is_menu_track_item(filter_value_1) or not filter_value_1`) and still have things work as intended?

Reply:
This has been reworked and is much cleaner now in commit Rework "if not x or y" logic 6d8faf3.  Good idea thank you.